Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Driver/Version2/IndexerDriver.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\Version2;
+
+/*
+ * This file is part of the Flowpack.ElasticSearch.ContentRepositoryAdaptor package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\Version1;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Mapping\NodeTypeMappingBuilder;
+use Flowpack\ElasticSearch\Domain\Model\Document as ElasticSearchDocument;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * Fulltext Indexer Driver for Elastic version 2.x
+ *
+ * @Flow\Scope("singleton")
+ */
+class IndexerDriver extends Version1\IndexerDriver
```

Comment:
@kdambekalns Can you check/port your recent fixes for the Groovy script in the 2.x driver ?